### PR TITLE
Make Travis use their new infra.

### PR DESCRIPTION
**Pull request by vmarmol**
/cc @rjnagal lets see if this works.

### CODE REVIEW

**Comment by vmarmol** [comment]
Woah, much faster :D

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: go
+sudo: false
```
Comment (rjnagal) [inline .travis.yml]: what's the new infra?

Comment (rjnagal) [inline .travis.yml]: ah, I see it now.

File: .travis.yml [hunk 2]
```diff
@@ -6,9 +7,6 @@ install:
 before_script:
  - go get golang.org/x/tools/cmd/vet
  - go get github.com/kr/godep
- - wget http://s3.amazonaws.com/influxdb/influxdb_latest_amd64.deb
- - sudo dpkg -i influxdb_latest_amd64.deb
- - sudo service influxdb start
```
Comment (rjnagal) [inline .travis.yml]: We don't need to test influxdb anymore?

Comment (vmarmol) [inline .travis.yml]: We haven't run these tests in a while. I've been meaning to remove them.

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**